From 740314db356b8d9f5ccf79d4613be225cc910f5a Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Mon, 22 Feb 2016 22:34:55 -0500
Subject: [PATCH] wr-policy: fix service radiusd

Reference sources: selinux-policy-3.13.1-128.21.fc22.src.rpm
Changes come from: policy-f22-base.patch

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 policy/modules/kernel/corenetwork.if.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/modules/kernel/corenetwork.if.in b/policy/modules/kernel/corenetwork.if.in
index 07126bd..88a0c04 100644
--- a/policy/modules/kernel/corenetwork.if.in
+++ b/policy/modules/kernel/corenetwork.if.in
@@ -1292,11 +1292,11 @@ interface(`corenet_dontaudit_tcp_bind_generic_port',`
 #
 interface(`corenet_udp_bind_generic_port',`
 	gen_require(`
-		type port_t;
+		type port_t, unreserved_port_t;
 		attribute defined_port_type;
 	')
 
-	allow $1 port_t:udp_socket name_bind;
+	allow $1 { port_t unreserved_port_t }:udp_socket name_bind;
 	dontaudit $1 defined_port_type:udp_socket name_bind;
 ')
 
-- 
1.9.1

